docs/test-configuration-options.md

```diff
@@ -11,80 +11,127 @@ export const Highlight = ({children, color}) => ( <span style={{
       padding: '0.2rem',
     }}>{children}</span> );
 
-In this topic, you'll find a list of Sauce Labs test configuration options. You can use the [Sauce Labs Platform Configurator](https://wiki.saucelabs.com/display/DOCS/Platform+Configurator) to generate the correct configuration of testing options for your choice of Appium or Selenium tests in your preferred programming language.
+This page includes a list of valid Sauce Labs test configuration options. 
+See the [Sauce Labs Platform Configurator](https://wiki.saucelabs.com/display/DOCS/Platform+Configurator) 
+for the basic code necessary for executing a test. Those settings as well as all valid parameters are described below.
 
-We always recommend using the latest released version of Selenium, but to ensure W3C WebDriver compliance, make sure to use Selenium version 3.11 or later. Sauce Labs determines W3C sessions with the presence of `sauce:options` capabilities and [generic W3C WebDriver-compliant capabilities](https://www.w3.org/TR/webdriver1/#capabilities). See [W3C Capabilities Support](https://wiki.saucelabs.com/pages/viewpage.action?pageId=78414463) for more information.
+We always recommend using the latest released version of Selenium, 
+but to ensure W3C WebDriver compliance, make sure to use Selenium version 3.11 or later. 
+Sauce Labs determines W3C sessions with the presence of `sauce:options` capabilities 
+and [generic W3C WebDriver-compliant capabilities](https://www.w3.org/TR/webdriver1/#capabilities). 
+See [W3C Capabilities Support](https://wiki.saucelabs.com/pages/viewpage.action?pageId=78414463) for more information.
 
 
-## Desktop Browser W3C Capabilities (Required)
+## WebDriver W3C Capabilities (Required)
 
-Below are the W3C WebDriver primary test configuration settings for Sauce Labs desktop browser tests.
+Below are the W3C WebDriver primary test configuration settings for Sauce Labs desktop browser tests and mobile tests.
 
-### `browserName`
-__Description__: identifies the user agent.
+### [`browserName`](https://w3c.github.io/webdriver/#dfn-browser-name)
```

Comment:
hi @titusfortner , linking this H3 breaks its `code` font in the sidebar. I tried playing around with it but couldn't get it to work. Do you have any suggestions?

![Screen Shot 2021-05-23 at 10 41 52 PM](https://user-images.githubusercontent.com/56411016/119301981-30450a00-bc18-11eb-801a-d687e921ca41.jpg)

Reply:
Yeah, we should probably pull it out and have the link be in the description.